src/library.js

```diff
@@ -3267,6 +3267,40 @@ LibraryManager.library = {
   },
   emscripten_asm_const_double: 'emscripten_asm_const_int',
 
+  $JsValStore: {
+    values: {},
+    next_id: 1,
+
+    add: function(js_val) {
+      var id;
+      do {
+        id = JsValStore.next_id++;
+        if (JsValStore.next_id > 2147483647) JsValStore.next_id = 1; // Wraparound signed int32.
+      } while (id in JsValStore.values);
+
+      JsValStore.values[id] = js_val;
+      return id;
+    },
+    remove: function(id) {
+#if ASSERTIONS
+      assert(id in JsValStore.values);
+#endif
+      delete JsValStore.values[id];
+    },
+    get: function(id) {
+#if ASSERTIONS
+      assert(id === 0 || id in JsValStore.values);
+#endif
+      return JsValStore.values[id];
+    },
+  },
+
+  emscripten_unwrap_js_handle__sig: 'ii',
```

Comment:
I think this should be `vi` since it returns `void`.
```suggestion
  emscripten_unwrap_js_handle__sig: 'vi',
```

Reply:
(in this case I think that signature can be deleted, as there is nothing aliasing/aliased to this, and it is not a proxied function either)

Reply:
Removed